fastlane/Fastfile

```diff
@@ -73,6 +125,12 @@ lane :update_native_dependencies_to_latest do |options|
   update_android_native_version(new_version: latest_android_release, current_version: current_android_version)
 
   run_pod_install
+
+  new_versions = {
+    new_ios_version: latest_ios_release,
+    new_android_version: latest_android_release
+  }
+  new_versions
```

Comment:
Hmm what's the reason to return this hash? Is this lane going to be used from a different lane?

Reply:
It's used in https://github.com/RevenueCat/purchases-hybrid-common/pull/179/files/f007a2a475bfda2d75d463cb1e1109b255f7c6aa#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R70